config/routes.rb

```diff
@@ -28,4 +28,13 @@
   get "auth/:provider/callback", to: "sessions#omniauth_callback"
 
   get "/dashboard", to: "main#dashboard", as: "dashboard"
+
+  resources :dynamic_pages, except: [:edit] do
```

Comment:
I think you could leave the edit route in, and don't need to use the one below, but this is fine as is. :)